resolver/resolver.cc

```diff
@@ -552,46 +552,61 @@ class ResolveConstantsWalk {
                     gs.beginError(core::Loc(todo.file, send->loc), core::errors::Resolver::InvalidMixinDeclaration)) {
                 e.setHeader("`{}` can only be declared inside a module, not a class", send->fun.data(gs)->show(gs));
             }
-            // Keep processing it anyways
+            return;
         }
 
-        if (send->args.size() != 1) {
+        if (send->args.size() < 1) {
             // The arity mismatch error will be emitted later by infer.
             return;
         }
-        auto &front = send->args.front();
-        auto *id = ast::cast_tree<ast::ConstantLit>(front);
-        if (id == nullptr || !id->symbol.exists() || !id->symbol.data(gs)->isClassOrModule()) {
-            if (auto e =
-                    gs.beginError(core::Loc(todo.file, send->loc), core::errors::Resolver::InvalidMixinDeclaration)) {
-                e.setHeader("Argument to `{}` must be statically resolvable to a module", send->fun.data(gs)->show(gs));
+
+        auto argumentNumber = 0;
+        auto encounteredError = false;
+        for (auto &arg : send->args) {
+            auto *id = ast::cast_tree<ast::ConstantLit>(arg);
+
+            if (id == nullptr || !id->symbol.exists() || !id->symbol.data(gs)->isClassOrModule()) {
+                if (auto e = gs.beginError(core::Loc(todo.file, send->loc),
```

Comment:
```suggestion
                auto loc = id != nullptr ? id->loc : send->loc;
                if (auto e = gs.beginError(core::Loc(todo.file, loc),
```
You could prefer the id loc when it exists. Additionally, it might be worth calling out the `id == nullptr` case as a separate error.

Reply:
I opted for a difference case for `id == nullptr` but noticed the tests were not testing the rest of the conditional `!id->symbol.exists() || !id->symbol.data(gs)->isClassOrModule()`. Added a new test but couldn't think of a good error message to differentiate the two cases.